src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+// Actions
+const ADD_BOOK = 'bookstore/books/ADD';
+const REMOVE_BOOK = 'bookstore/books/REMOVE';
```

Comment:
- [ ] @SSEKPIUS great work on your reducer, however in this case the action type definition doesn't follow the[ ducks pattern](https://github.com/erikras/ducks-modular-redux), the last name in your case 'ADD' should be equal to the const in your case 'ADD_BOOK'